jormungandr/src/network/p2p/topology.rs

```diff
@@ -29,13 +29,15 @@ impl From<bincode::Error> for Error {
     }
 }
 
-#[derive(Clone, Debug)]
 pub struct Node(poldercast::Node);
 
+#[derive(Clone, Debug)]
+pub struct NodeData(poldercast::NodeData);
```

Comment:
Losing track a bit as to why there is now `Node` and `NodeData` when the useful things are now done using the latter. Some doc comments would be nice.

Reply:
I understand, I am differentiating the `Node`, ourselves, from the `Data` that we have on other `Node` (or ourselves).

So now, locally everyone has a `Node` and a set of `NodeData`. The `Node` contains more info than before so adding an object to superseed the `NodeData` was necessary (i.e. we don't want to gossip private keys for example).

Reply:
I see. Maybe a name like `SelfNode` would help understand the difference.